src/main/java/vazkii/botania/api/mana/ILensEffect.java

```diff
@@ -49,7 +48,7 @@
 	/**
 	 * Gets the amount of mana to transfer to the passed in mana receiver block.
 	 */
-	public default int getManaToTransfer(IManaBurst burst, ThrowableEntity entity, ItemStack stack, IManaReceiver receiver) {
+	public default int getManaToTransfer(IManaBurst burst, ItemStack stack, IManaReceiver receiver) {
```

Comment:
isn't this an api break

Reply:
i mean technically yes but are there really any 1.16.3 botania addons right now